packages/components/src/tabs/tablist.tsx

```diff
@@ -64,18 +79,19 @@ export const TabList = forwardRef<
 
 	return (
 		<Ariakit.TabList
-			ref={ ref }
+			ref={ refs }
 			store={ store }
 			render={
 				<TabListWrapper
 					onTransitionEnd={ ( event ) => {
-						if ( event.pseudoElement === '::after' ) {
+						if ( event.pseudoElement === '::before' ) {
 							setAnimationEnabled( false );
 						}
 					} }
 				/>
 			}
 			onBlur={ onBlur }
+			tabIndex={ -1 }
```

Comment:
This is in place to prevent Firefox from making it focusable by keyboard, which is a behavior that happens automatically when there's overflow in `overflow: auto` containers (see https://github.com/WordPress/gutenberg/pull/64371#issuecomment-2330979556). Tested and it works to prevent that behavior.